types/passport-local-mongoose/index.d.ts

```diff
@@ -1,119 +1,118 @@
-// Type definitions for passport-local-mongoose 4.0.0
+// Type definitions for passport-local-mongoose 6.1
 // Project: https://github.com/saintedlama/passport-local-mongoose
 // Definitions by: Linus Brolin <https://github.com/linusbrolin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.0
-
-/// <reference types="mongoose" />
-/// <reference types="passport-local" />
+// TypeScript Version: 4.0
```

Comment:
```suggestion

/// <reference types="mongoose" />

```
not sure why this would require TS4 or newer
(restore previous ref for agmented module)

Reply:
Thanks again.  I restored the missing types reference including 'passport-local' and lowered TypeScript to 3.7.  I couldn't got lower without getting a circular type error from the compiler.